src/index.js

```diff
@@ -1,78 +1,106 @@
-// This is the JavaScript entry file - your code begins here
-// Do not delete or rename this file ********
-
-// An example of how you tell webpack to use a CSS (SCSS) file
 import './css/base.scss';
 
-// An example of how you tell webpack to use an image (also need to link to it in the index.html)
-import './images/turing-logo.png'
-
 import ApiFetch from './ApiFetch';
-import domUpdates from './domUpdates';
 import Traveler from './traveler';
 
-console.log('This is the JavaScript entry file - your code begins here.');
+let userID, destinationData;
 
 window.addEventListener('load', onLoad);
 
-
 function onLoad() {
   const loginBtn = document.querySelector('.login-button')
   loginBtn.addEventListener('click', enterLogin)
-
-  // displayPastTrips()
-
+  const bookNewTripBtn = document.querySelector('.new-trip-button')
+  bookNewTripBtn.addEventListener('click', postTrip)
 }
 
 function enterLogin() {
   const username = document.getElementById('username').value
   const password = document.getElementById('password').value
-  const loginValue = parseInt(username.match(/\d+/gi).pop())
-  console.log("VALUE", loginValue)
+  userID = parseInt(username.match(/\d+/gi).pop())
   const removeLogin = document.querySelector('.entry-form')
+  const addForm = document.querySelector('.trip-entry-form')
+  const text = document.querySelector('.scroll')
   if (username.includes('traveler') && (username.split('traveler')[1] < 51 && username.split('traveler')[1] > 0) && password === 'travel2020') {
     console.log(username + ' is logged in!!!')
     removeLogin.classList.add('hidden')
-    getData(loginValue)
+    addForm.classList.remove('hidden')
+    addForm.classList.add('flex')
+    text.classList.remove('hidden')
+    getData(userID)
   } else {
     alert("WRONG PASSWORD")
   }
-
 }
 
-//get the value of username
-
-// var str = "traveler50";
-// var res = str.match(/\d+/gi);
-
-
 function getData(loginValue) {
   const api = new ApiFetch();
   const tripData = api.getTripsData()
   const travelerData = api.getTravelersData()
-  console.log("TD", travelerData)
   const destinationData = api.getDestinationsData();
   function _createTraveler(data) {
-    createTraveler(data, loginValue)
+    return createTraveler(data, loginValue)
   }
+  Promise.all([travelerData, tripData, destinationData])
+    .then(data => data = {
+      travelerData: data[0].travelers,
+      tripData: data[1].trips,
+      destinationData: data[2].destinations
+    })
+    .then(_createTraveler)
+    .then(displayPastHTML)
+    .catch(error => console.log(error));
+
+  Promise.all([travelerData, tripData, destinationData])
+      .then(data => data = {
+        travelerData: data[0].travelers,
+        tripData: data[1].trips,
+        destinationData: data[2].destinations
+      })
+      .then(fillDestinations)
+      .catch(error => console.log(error));
+}
```

Comment:
Wonder if it would have been possible to compress the 2nd `Promise.all()` by using another `.then` on the other `Promise.all()`. Great to see how concise your promise.all are without doing additional logic inside it.